dockers/base-conda/Dockerfile

```diff
@@ -29,7 +29,10 @@ ENV \
     # CUDA_TOOLKIT_ROOT_DIR="/usr/local/cuda" \
     MKL_THREADING_LAYER=GNU
 
-RUN apt-get update -qq --fix-missing && \
+RUN \
+    # https://github.com/NVIDIA/nvidia-docker/issues/1631
```

Comment:
@akihironitta Let's add a TODO here. So that we could update it if the change is upstreamed to the official distro

Reply:
Done! Thank you for your suggestion!